main.go

```diff
@@ -832,7 +866,11 @@ func (p *gateway) start(ctx context.Context) error {
 	// setup scheduler
 	scheduler := p.setupScheduler(p.config)
 
-	if err := p.setupDebugServer(p.config, p.logger, scheduler); err != nil {
+	// set up defaultDelayScheduler
+	defaultDelayScheduler := p.setupScheduler(p.config)
```

Comment:
can't use the same setup routine, it includes the go metrics source stuff, we don't want that